boards/arm/96b_avenger96/doc/index.rst

```diff
@@ -185,7 +185,8 @@ Default Zephyr Peripheral Mapping:
 System Clock
 ------------
 
-The Cortex®-M4 Core is configured to run at a 209 MHz clock speed.
+The Cortex®-M4 Core is configured to run at a 209 MHz clock speed. This value
+should match the configured mlhclk_ck frequency.
```

Comment:
should this be ``must match`` (is it OK if it doesn't?)

Reply:
it must indeed, txs